programming-languages/Python/2-curriculum/2-installing-python/0-introduction.md

```diff
@@ -1,3 +1,6 @@
 <h3 align="center"> Installing python </h3>
 Topics Covered - Installing python in windows, Mac and Linux
 Note - Python3 is recommended
+
+
+If you want to use a different version, otherthan the recommended version Python3, then it is suggested to use Anaconda or miniconda. You can create your own environments accordingly based on your application.
```

Comment:
@Abhishekmamidi123 please correct the typo in otherthan to other than. But still note that all of the suggestions that I have made are on python3.

Reply:
It's better to mention the learners to use Anaconda or miniconda even if Python3 is recommended